gui.py

```diff
@@ -357,12 +408,18 @@ def __get_kontaktdaten(self, modus: Modus) -> dict:
             self.kontaktdaten_erstellen(modus)
 
         kontaktdaten = kontak_tools.get_kontaktdaten(self.pfad_kontaktdaten)
-        if not self.__check_old_kontakt_version(kontaktdaten):
-            raise ValidationError("\"zeitrahmen\" fehlt -> Alte Version")
+        if modus == Modus.TERMIN_SUCHEN:
```

Comment:
Hier noch einen check der Kontaktdaten hinzufügen, um sicher zu gehen, dass alle Kontakdaten auch vollständig sind

```suggestion
        kontak_tools.check_kontaktdaten(kontaktdaten, modus)
        
        if modus == Modus.TERMIN_SUCHEN:
```